Sift/Schema/ComplexTypes/browser.json

```diff
@@ -5,6 +5,12 @@
   "properties": {
     "$user_agent": {
       "type": [ "string", "null" ]
+    },
+    "$accept_language": {
```

Comment:
If you're feeling brave, could be a good chance to write tests for `browser` as well.

Reply:
done